### ISSUE TITLE
Canvas name hardcoded in AndroidProject

### ISSUE THREAD

**Issue by DariosKrimsKrams** [peterhorsley/Unity3D.Authentication.Example#5]
The Canvas name is hardcoded in the AndroidProject: To use the Activity behind the plugin with your own project (your own scenes/canvas), you need to rename your Canvas to the same name used in demo project. Better solution would be to changed it in AndroidProject but then you have to re-build the AndroidUnityPlugin.jar file.
  * At least a info in the documentation is needed.
  * A better solution for this problem would be nice.
  * I don't actually know if there is a problem when using this with multiple canvas,, but I could expect this doesn't work at the moment :/

Edit: Realized there is already a issue for that: https://github.com/peterhorsley/Unity3D.Authentication.Example/issues/1

**Comment by DarioDomiDE** [comment]
Edit: I mean "SignInCanvas" name, not scene name. Realized there is already a issue for that: https://github.com/peterhorsley/Unity3D.Authentication.Example/issues/1

**Comment by peterhorsley** [comment]
Agree, a better solution would be nice, but adds some complexity i think.  For now I just updated the readme to document this requirement, I guess you could consider it configuration by convention for now.
